src/ts-framework-tests/app/controllers/ShopController.ts

```diff
@@ -1,19 +1,23 @@
 import {Controller, action} from "../../../ts-framework/Controller";
 import {JsonResult} from "../../../ts-framework/View";
+import {HttpController} from "../../../ts-framework/Controller/HttpController";
 
-export class ShopController extends Controller
+export class ShopController extends HttpController
 {
     // GET: /shop
     @action() index()
     {
-        this.response.sendContent("Hello");
+        this.content("Hello");
```

Comment:
suggestion: `new ContentResult()` It seems more pleasant, besides leaving cleaner HttpController